lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -802,6 +802,37 @@ def test_unenroll_non_user_student(
             ).exists()
         )
 
+    @patch('ccx.views.render_to_response', intercept_renderer)
+    @ddt.data(
+        ('ccx_invite', 'student-ids', ('enrollment-button', 'Enroll'), 'nobody@nowhere.com'),
+        ('ccx_manage_student', 'student-id', ('student-action', 'add'), 'nobody2@nowhere.com'),
+    )
+    @ddt.unpack
+    def test_enrollment_status(self, view_name, student_form_input_name, button_tuple, identifier):
+        """
+        Get enrollment status in the list of management student on CCX coach dashboard.
+        """
+        self.make_coach()
+        ccx = self.make_ccx()
+        course_key = CCXLocator.from_course_locator(self.course.id, ccx.id)
+        ccx_coach_url = reverse('ccx_coach_dashboard', kwargs={'course_id': course_key})
+        url = reverse(
+            view_name,
+            kwargs={'course_id': course_key}
+        )
+
+        data = {
+            button_tuple[0]: button_tuple[1],
+            student_form_input_name: u','.join([identifier, ]),
```

Comment:
@lukyfebrianto why do you need join here?

Reply:
To handle enroll student from using batch enrollment,
![screenshot from 2017-11-16 15-56-02](https://user-images.githubusercontent.com/25189966/32879906-b8aea986-cae6-11e7-9645-b6bd6297b9fa.png)

Reply:
Please check the following result:
```py
In [1]: ','.join(['test'])
Out[1]: 'test'
```
in this case `identifier = 'test'`. Why do we need to join a single element?

Reply:
ahh yes, removed